setup.py

```diff
@@ -75,6 +75,7 @@
     extras_require['all'].extend(reqs)
 
 install_requires = [
+    'arrow>=0.8.0',
```

Comment:
alternative is to ``import calendar`` and use https://docs.python.org/2/library/calendar.html#calendar.timegm, but arrow seemed to be more abundant across other packages, so I'd vote yes for the consistency.